script.maps.browser/resources/language/English/strings.po

```diff
@@ -0,0 +1,651 @@
+# XBMC Media Center language file
+# Addon Name: Maps Browser
+# Addon id: script.maps.browser
+# Addon version: 0.8.0
+# Addon Provider: phil65
+msgid ""
+msgstr ""
+"Project-Id-Version: XBMC-Addons\n"
+"Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
+"POT-Creation-Date: 2014-10-26 17:04+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+
+msgctxt "Addon Summary"
+msgid "Google Maps Browser (including Street View)"
+msgstr ""
+
+msgctxt "Addon Description"
+msgid ""
+"This Maps Browser Add-on allows you to explore the map with a remote. It can also show Places (Source: FourSquare), "
+"events (source: Eventful), geotagged images and much more on the map. Also offers skin integration."
```

Comment:
not sure if it's valid like this, but perhaps stick with using a single line:
see: https://github.com/xbmc/xbmc/commit/5e7507e663461e023aec4a958a6d6dadaea493b9